ethereum/spec/src/main/java/tech/pegasys/teku/spec/signatures/SigningRootUtil.java

```diff
@@ -117,4 +117,13 @@ public Bytes signingRootForValidatorRegistration(
     final Bytes32 domain = miscHelpers.computeDomain(Domain.APPLICATION_BUILDER);
     return miscHelpers.computeSigningRoot(validatorRegistration, domain);
   }
+
+  /* overloaded method that uses the instance Spec instead of epoch to calculate the signing. Used
+  by non-Teku clients of this library. */
+  public Bytes signingRootForValidatorRegistration(
```

Comment:
I'd be more incline to use the same method but with an `Optional` epoch to avoid code duplication.
and then use `spec.getGenesisSpec().miscHelpers()` to avoid creating a new instance.

Reply:
👍🏼 I'll incorporate it.